herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -78,15 +90,48 @@
                         error('Detector indentifiers must all be unique')
```

Comment:
I know this is old code, but worth going through and adding error identifiers?

Reply:
after the merge, yes